main.js

```diff
@@ -0,0 +1,101 @@
+/*eslint-disable */
+class Book {
+    constructor (title,author) {
+        this.title = title;
+        this.author = author;
+    }
+}
+
+
+/*eslint-disable */
+class UI {
+    
+    static displayBook() {
+        const books = store.getBooks();
+    
+
+        books.forEach((book) => UI.addBookToList(book));
+    }
+    static addBookToList(book){
+        const list = document.querySelector('#book-list');
+        const div = document.createElement('div');
+        div.innerHTML = `
+        <div>${book.title}</div>
+        <div>${book.author}</div>
+        <button class='delete' id='delete'>Remove</button> 
+        `;
+        list.appendChild(div);
+        console.log(book.title);
+        console.log(book.author);
+    }
+   
+
+    static deleteBook(clickTarget){
+        if(clickTarget.classList.contains('delete')){
+            clickTarget.parentElement.remove();
+        }
+    }
+    static clearFields(){
+        document.querySelector('#book').value ='';
+        document.querySelector('#author').value ='';
+    }
+}
+/*eslint-disable */
+
+/*eslint-disable */
+class store {
+     static getBooks() {
+        let books;
+        if(localStorage.getItem('books') === null){
+            books = [];
+        }else{
+            books = JSON.parse(localStorage.getItem('books'));
+        }
+        return books;
+    }
+
+    static addBook(book) {
+        const books = store.getBooks();
+        books.push(book);
+        localStorage.setItem('books', JSON.stringify(books))
+    }
+
+    static removeBook(author) {
+        const books = store.getBooks();
+        books.forEach((book,index) => {
+            if (book.author === author ){
+                books.splice(index,1)
+                localStorage.setItem('books',JSON.stringify(books))
+            }
+        })
+    }
+}
+/*eslint-disable */
+
+// event  displaybook
+
+document.addEventListener('DOMContentLoaded',UI.displayBook);
+
+document.querySelector('#booksform').addEventListener('submit',(e) =>{
+    e.preventDefault();
+    const title = document.querySelector('#book').value;
+    const author = document.querySelector('#author').value;
+    console.log( document.querySelector('#author').value)
```

Comment:
- [ ] Consider removing all `console.log` calls, they're useful when you're trying to debug your code, but they don't serve any functionality to the feature and they pollute your codebase. Kindly remove them! 🤓 🛠️